embassy-stm32/src/adc/v2.rs

```diff
@@ -150,7 +153,7 @@ impl Prescaler {
 
 pub struct Adc<'d, T: Instance> {
     sample_time: SampleTime,
-    calibrated_vdda: u32,
+    vref: u32,
```

Comment:
if we have no typed struct for mV it'd be good to specify the unit in the field name: `vref_mv`.